src/main.rs

```diff
@@ -44,5 +49,6 @@ fn main() {
         .add_plugin(PlayerPlugin)
         .add_plugin(UiPlugin)
         .add_plugin(WorldPlugin)
+        .add_plugin(MenuPlugin)
```

Comment:
Should MenuPlugin be added to src/ui/mod.rs instead?

Reply:
Yes. Good point.